### ISSUE TITLE
Question about Archived downloads slipping back into the Backlog Manager

### ISSUE THREAD

**Issue by LlmDl** [SickChill/SickChill#5292]
Branch: master
Commit: f7a113a737654a91076bb42221e119db6258a276
Version: v2019.01.06-1
OS: Windows-7-6.1.7601-SP1
Browser: Vivaldi 
What you did:
My Backlog Manager was full of episodes that were set to Allowed. I took all of my shows/seasons that did not match their desired Resolution and set them to Archived. 

What happened:
This emptied the backlog manager but every day a show or two has a season slip back into their Allowed lists.

What you expected:
That the Backlog Manager list would not grow and the Archived shows would not fall into the Allowed list. 

I'm mostly just posting this to see if this is expected behaviour or if there's a setting I need to find/deactivate in order to have my Backlog Manager stay fairly clean of older downloads.

**Comment by VonSchnitzel** [comment]
same question here. No matter what I set episodes to, skipped/archived etc. the same ones keep reappearing in the backlog search.

**Comment by WebSpider** [comment]
Archived episodes are supposed to be ignored. Please attach some debug logs if you can find when it is being added to searched again

**Comment by ryangribble** [comment]
This is happening to me as well.  I believe it happens overnight in the daily show refresh.  It can be manually reproduced by performing "rescan files" or "force full refresh" options on the show in question.

Not sure what's happened as no changes have occurred to SickChill since 9th January, and yet just 3 days ago my setup suddenly downloaded 200 old backlogged episodes, and my backlog overview showed lots of yellow coloured "allowed" status episodes.  I changed them all manually to Archived (Unknown), but then the next day at least some were reverted back to Downloaded (Unknown) again.

A number of the problematic episodes in my case turned out to have duplicate files for whatever reason, eg a file had been properly downloaded, renamed/copied into the show directory, then the show directory also had an un-renamed file for the same episode in it, or sometimes in the parent show folder rather than the season folder.

In another case, there was no duplicate file, but that was American Dad! which has very misaligned scene/torrent numbering etc, so Im not surprised if things are not working too well on American Dad!


However I do think that expected behaviour should be that there is a way to flag an episode to be ignored so that even a rescan of files won't change it's status back to something which causes it to be added to backlog list and downloaded.  It is also strange because there havent been any changes in over a week, yet this suddenly started happening only 3 days ago
